vignettes/articles/scripting-tasks-config.Rmd

```diff
@@ -0,0 +1,477 @@
+# `task_ids_example`
+task_ids_example <- create_task_ids(origin_date_id, location_id, horizon_id, target_id)
+
+```
+
+Now that we've created the modeling parameters, we can define the outputs we
+expect from the models. 
+
+### Creating the `output_type` objects
+
+For this example, we want to have three output type objects: a required "mean"
+and "quantile" outputs with an optional integer "median".
+
+```{r create-output-type-ids}
+# Create a representation of a `mean` output type as a list object of class
+# `output_type_item`
+mean_example <- create_output_type_mean(
+  is_required = TRUE,
+  value_type = "double",
+  value_minimum = 0
+)
+
+# Create a representation of a `median` output type as a list object of class
+# `output_type_item`
+median_example <- create_output_type_median(
+  is_required = FALSE,
+  value_type = "integer",
+  value_minimum = 0L
+
+)
+
+# Create a representation of a `quantile` output type as a list object of class `output_type_item`
+quantile_example <- create_output_type_quantile(
+  required = c(0.25, 0.5, 0.75),
+  is_required = TRUE,
+  value_type = "double",
+  value_minimum = 0
+)
+```
+
+Now that we have our base output types, we can combine them to create output
+types that we want to use for each particular task. 
+
+```{r combine-output-types}
+# Combine the point estimate `output_type_item` objects to create an
+# `output_type` class object, the `output_type_mean_median`
+output_type_mean_median <- create_output_type(mean_example, median_example)
+
+# Combine the other `output_type_item` objects to create an `output_type` class
+# object, the `output_type_mean_quantile`
+output_type_mean_quantile <- create_output_type(mean_example, quantile_example)
+```
+
+### Creating the `model_task` objects
+
+For both tasks, we want to require a "mean" output, but we want a "quantile"
+output type for the `inc death` target and an optional "median" for the `inc
+hosp` target.
+
+```{r create-model-task}
+#Create an object of class `model_task` representing a model task, the `model_task_hosp`
+model_task_hosp <- create_model_task(
+  task_ids = task_ids_example,
+  output_type = output_type_mean_median,
+  target_metadata = target_metadata_example
+)
+
+#Create another object of class `model_task` representing a similar model task, the `model_task_death`
+model_task_death <- create_model_task(
+  task_ids = task_ids_example,
+  output_type = output_type_mean_quantile,
+  target_metadata = target_metadata_example
+)
+
+# Combine the `model_task` objects to create a `model_tasks` class object, the
+# `model_task_example`
+model_task_example <- create_model_tasks(model_task_hosp, model_task_death)
+```
+Now that we have a set of model tasks, we can create rounds for them.
+
+### Creating the `round` objects
+
+While the model tasks define what a model output submission should look like,
+the round defines the cadence of submissions and the timeframe under which a
```

Comment:
```suggestion
the round additionally defines the schedule of submissions and the timeframe under which a
```
_additionally_ because it also contains all the info contained in the model tasks and using a more lay person word instead of cadence